manoseimas/mps_v2/models.py

```diff
@@ -74,16 +75,16 @@ def __unicode__(self):
 
 class Group(CrawledItem):
     GROUP_TYPES = (
-        ('group', ('Group')),
-        ('committee', ('Committee')),
-        ('commission', ('Commission')),
-        ('fraction', ('Fraction')),
-        ('parliament', ('Parliament')),
+        ('group', _('Group')),
+        ('committee', _('Committee')),
+        ('commission', _('Commission')),
+        ('fraction', _('Fraction')),
+        ('parliament', _('Parliament')),
     )
 
-    name = models.CharField(max_length=128, unique=True)
+    name = models.CharField(max_length=255, unique=True)
     slug = AutoSlugField(populate_from='name')
-    type = models.CharField(max_length=16,
+    type = models.CharField(max_length=64,
                             choices=GROUP_TYPES)
```

Comment:
I'd prefer to keep it at 80 because my workflow often involves vertical splits. I'm ok with it being 120 in some places.